src/components/URLImportDialog/index.js

```diff
@@ -0,0 +1,255 @@
+import React, { useState } from "react";
+
+import { makeStyles } from "@material-ui/core/styles";
+import { useTheme } from "@material-ui/core/styles";
+import useMediaQuery from "@material-ui/core/useMediaQuery";
+
+import UploadIcon from "mdi-react/UploadIcon";
```

Comment:
I'm not sure about importing another icon library when MUI already provides many of the official material.io icons.

Although I had trouble finding one that looked better or similar to this one using https://material.io/resources/icons/, I did find a similar `Publish` icon with https://material-ui.com/components/material-icons/.